shuffle-plugin/src/main/scala/com/nvidia/spark/rapids/shuffle/ucx/UCX.scala

```diff
@@ -85,7 +84,7 @@ class UCX(transport: UCXShuffleTransport, executor: BlockManagerId, rapidsConf:
 
   logInfo(s"UCX context created")
 
-  def getExecutorId: Int = executor.executorId.toInt
+  def getExecutorId: Long = executor.executorId.toLong
```

Comment:
since we are changing this line, this is a pretty inefficient transformation to be a def https://github.com/scala/scala/blob/v2.12.6/src/library/scala/collection/immutable/StringLike.scala#L306

Can we make it a val?